packages/definitions/dist/fhir/r4/valuesets-medplum.json

```diff
@@ -808,6 +808,35 @@
           }
         ]
       }
+    },
+    {
+      "fullUrl": "http://medplum.com/ValueSet/medplum-provider-lab-procedure-valueset",
```

Comment:
Hmm... We should think about this.

This file was really intended for Medplum-specific ValueSets that are referenced by the Medplum resource StructureDefinitions (such as Agent.status or Bot.runtimeVersion, etc)

This ValueSet looks more like an example.

I don't really have a better suggestion for where this belongs, so I'm not strictly opposed to it.  

Do we anticipate having a "provder app setup script" somewhere?  Like a medplum-ee tool that preps a project for the provider app?  That might be more appropriate?

Reply:
This has been removed and will be made available through a linked project.